src/solvers/flattening/bv_conversion_exceptions.h

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************\
+
+Module: Bit vector conversion
+
+Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+/// \file
+/// Exceptions that can be raised in bv_conversion
+
+#ifndef CPROVER_SOLVERS_FLATTENING_BV_CONVERSION_EXCEPTIONS_H
+#define CPROVER_SOLVERS_FLATTENING_BV_CONVERSION_EXCEPTIONS_H
+
+#include <util/arith_tools.h>
+#include <util/endianness_map.h>
+
+#include "boolbv.h"
+#include "bv_conversion_exceptions.h"
+#include "flatten_byte_extract_exceptions.h"
+#include "flatten_byte_operators.h"
```

Comment:
These includes just seem like nonsense - `bv_conversion_exceptions.h` must not be needed since that's the file we're in! Replaced with what appear to me to be correct